.github/CONTRIBUTING.md

```diff
@@ -19,3 +19,7 @@ All the issues related to ALMighty Core should be reported in the
 
 Your pull requests can be send to Github repository against the master
 branch.
+
+## Reviewing
+
+[This page](https://github.com/golang/go/wiki/CodeReviewComments) collects common comments made during reviews of Go code, so that a single detailed explanation can be referred to by shorthands. This is a laundry list of common mistakes, not a style guide.
```

Comment:
@kwk What about adding some additional links as references. 

go good practices and godoc in general. Those could be considered as upstream conventions. Ofc nothing written in a stone..

Effective Go: https://golang.org/doc/effective_go.html
Godoc: https://blog.golang.org/godoc-documenting-go-code. I guess we want the people to